### ISSUE TITLE
Multiple deprecation warnings when compiling project

### ISSUE THREAD

**Issue by chmike** [rejectedsoftware/vibe.d#1483]
I'm using vibe.d 0.7.28 and dmd v2.071.0.
```
XXX@home:~/Sources$ dub init testVibe -t vibe.d
Successfully created an empty project in '/home/XXX/Sources/testVibe'.
XXX@home:~/Sources$ cd testVibe/
XXX@home:~/Sources/testVibe$ dub build
Performing "debug" build using dmd for x86_64.
vibe-d:utils 0.7.28: target for configuration "library" is up to date.
vibe-d:data 0.7.28: target for configuration "library" is up to date.
vibe-d:core 0.7.28: target for configuration "libevent" is up to date.
vibe-d:http 0.7.28: target for configuration "library" is up to date.
vibe-d:diet 0.7.28: target for configuration "library" is up to date.
vibe-d:mail 0.7.28: target for configuration "library" is up to date.
vibe-d:mongodb 0.7.28: target for configuration "library" is up to date.
vibe-d:redis 0.7.28: target for configuration "library" is up to date.
vibe-d:web 0.7.28: target for configuration "library" is up to date.
vibe-d 0.7.28: target for configuration "libevent" is up to date.
testvibe ~master: building configuration "application"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(395,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(426,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
Linking...
To force a rebuild of up-to-date targets, run again with --force.
m
```

If I do a force rebuild it gets worse.

```
XXX@home:~/Sources/testVibe$ dub build --force
Performing "debug" build using dmd for x86_64.
vibe-d:utils 0.7.28: building configuration "library"...
vibe-d:data 0.7.28: building configuration "library"...
vibe-d:core 0.7.28: building configuration "libevent"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/core/core.d(1076,10): Deprecation: module std.concurrency is not accessible here, perhaps add 'static import std.concurrency;'
../../.dub/packages/vibe-d-0.7.28/source/vibe/core/log.d(788,18): Deprecation: module std.utf is not accessible here, perhaps add 'static import std.utf;'
../../.dub/packages/vibe-d-0.7.28/source/vibe/core/sync.d(186,11): Deprecation: vibe.inet.path.min is not visible from module sync
../../.dub/packages/vibe-d-0.7.28/source/vibe/stream/stdio.d(146,16): Deprecation: vibe.inet.path.min is not visible from module stdio
vibe-d:http 0.7.28: building configuration "library"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/http/common.d(618,7): Deprecation: vibe.stream.operations.isOutputRange is not visible from module common
../../.dub/packages/vibe-d-0.7.28/source/vibe/http/server.d(1267,22): Deprecation: vibe.inet.path.canFind is not visible from module server
../../.dub/packages/vibe-d-0.7.28/source/vibe/http/websockets.d(85,26): Deprecation: vibe.core.core.Tuple is not visible from module websockets
../../.dub/packages/vibe-d-0.7.28/source/vibe/http/websockets.d(103,21): Deprecation: vibe.core.core.tuple is not visible from module websockets
vibe-d:diet 0.7.28: building configuration "library"...
vibe-d:mail 0.7.28: building configuration "library"...
vibe-d:mongodb 0.7.28: building configuration "library"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/mongo/settings.d(89,12): Deprecation: vibe.data.serialization.to is not visible from module settings
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/mongo/settings.d(92,4): Deprecation: vibe.data.serialization.enforce is not visible from module settings
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/mongo/settings.d(128,12): Deprecation: vibe.data.serialization.to is not visible from module settings
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/mongo/settings.d(139,12): Deprecation: vibe.data.serialization.to is not visible from module settings
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/mongo/settings.d(169,21): Deprecation: vibe.data.serialization.to is not visible from module settings
vibe-d:redis 0.7.28: building configuration "library"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/redis/idioms.d(378,53): Deprecation: vibe.db.redis.types.seconds is not visible from module idioms
../../.dub/packages/vibe-d-0.7.28/source/vibe/db/redis/idioms.d(379,24): Deprecation: vibe.db.redis.types.msecs is not visible from module idioms
vibe-d:web 0.7.28: building configuration "library"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(395,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(426,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/validation.d(34,14): Deprecation: vibe.utils.validation.appender is not visible from module validation
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/validation.d(78,14): Deprecation: vibe.utils.validation.appender is not visible from module validation
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/validation.d(122,14): Deprecation: vibe.utils.validation.appender is not visible from module validation
vibe-d 0.7.28: building configuration "libevent"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(395,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(426,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
testvibe ~master: building configuration "application"...
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(395,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
../../.dub/packages/vibe-d-0.7.28/source/vibe/web/internal/rest/common.d(426,13): Deprecation: vibe.web.rest.HTTPMethod is not visible from module common
Linking...
XXX@home:~/Sources/testVibe$
```

**Comment by TooHuman** [comment]
I am getting the same, using DMD v2.071.0 and having ran "dub upgrade" running vibe-d-0.7.28.

**Comment by s-ludwig** [comment]
Using 0.7.29-beta.2 instead should have this fixed (use `dub upgrade --prerelease` to try).

**Comment by s-ludwig** [comment]
Fixed and published with 0.7.29.
